docs/user-guides/forwarding-logs-via-http/kubesphere/output-http.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: logging.kubesphere.io/v1alpha2
+kind: Output
+metadata:
+  name: errohttp
+  namespace: kubesphere-logging-system
+  labels:
+    logging.kubesphere.io/enabled: "true"
+    app.kubernetes.io/version: v0.2.0
+spec:
+  match: kube_erro
+  http:
+    # Change the host and port to your receiver endpoint
+    host: "log-receiver"
```

Comment:
host: "log-receiver" =>   host: "<log-receiver>"
Add a bracket to make it more clear that this is a param that should be changed?